frontend/src/modules/editor/contentObject/views/editorMenuView.js

```diff
@@ -82,8 +80,25 @@ define(function(require){
         _parentId: model.get('_id'),
         models: this.contentobjects.where({ _parentId: model.get('_id') })
       });
+      this.layerViews[model.get('_id')] = menuLayerView;
       $('.editor-menu-inner').append(menuLayerView.$el);
-      if(typeof callback === 'function') callback();
```

Comment:
I'm not sure we still need this async stuff? I may have left it in accidentally. Will also need to remove func param

Reply:
thanks, fixed it